src/db/tileDatabase.ts

```diff
@@ -0,0 +1,28 @@
+import * as sqlite3 from "sqlite3";
+import { TilesRow } from "../types/db";
+
```

Comment:
It would be helpful to document what information is expected to be in a tile database / in an individual tile_data record: saying whether it's shapes, indicators, or both.